bundles/net.tourbook.ext.jars/META-INF/MANIFEST.MF

```diff
@@ -19,13 +19,26 @@ Bundle-ClassPath:
  simplelatlng-1.3.1.jar,
  jackson-core-2.10.3.jar,
  jackson-databind-2.10.3.jar,
- jackson-annotations-2.10.3.jar
+ jackson-annotations-2.10.3.jar,
+ dropbox-core-sdk-3.1.3.jar,
+ java7-fs-dropbox-0.0.3.jar,
+ java7-fs-base-0.0.3.jar,
+ httpcore-4.4.13.jar,
+ httpclient-4.5.12.jar
 Export-Package: 
+ com.dropbox.core,
+ com.dropbox.core.oauth,
+ com.dropbox.core.util,
+ com.dropbox.core.v2,
+ com.dropbox.core.v2.auth,
+ com.dropbox.core.v2.files,
```

Comment:
As we discussed in the mail from 13.04.2020, the dropbox feature should be optional, MT should not have any dependencies on it. The implementation of a device data reader could be a blue print e.g. the fit data reader plugin could be removed (or disabled in the Eclipse IDE) and MT is still working without any error except that it cannot import .fit files and the fit import pref page is not displayed.

Currently only the drop box pref page is in the clould plugin and not all the other dropbox stuff.

I know that you may not happy with this review but I want that MT is still a plugable app (when possible) and this feature would be a good example.